easybuild/easyconfigs/b/bcl2fastq2/bcl2fastq2-2.20.0-GCC-11.2.0.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'CMakeMake'
+
+name = 'bcl2fastq2'
+version = '2.20.0'
+
+homepage = 'https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html'
+description = """bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by
+ Illumina sequencing systems to standard FASTQ file formats for downstream analysis."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True, 'cstd': 'c++11'}
+
+source_urls = ['ftp://webdata2:webdata2@ussd-ftp.illumina.com/downloads/software/bcl2fastq/']
+sources = [{
+    'filename': '%s-v%s-tar.zip' % (name, version.replace('.', '-')),
+    'extract_cmd': 'unzip -p %s | tar -xzvf -',  # source file is a .zip that contains a .tar.gz
+}]
+patches = [
+    '%(name)s-%(version)s-fix-cmake-target-libs.patch',
+    # bcl2fastq2 v2.20.0 is not compatible with recent Boost versions, use bundled Boost
+    '%(name)s-%(version)s-find-boost.patch',
+]
+checksums = [
+    '8dd3044767d044aa4ce46de0de562b111c44e5b8b7348e04e665eb1b4f101fe3',  # bcl2fastq2-v2-20-0-tar.zip
+    '7908c268c213a6cfeafe05afcd5d3b34098241b6ab0a62c11ea47664cd0a63c5',  # bcl2fastq2-2.20.0-fix-cmake-target-libs.patch
+    '8ca9ab8843fd21976781185b0d560f97fa83c26ab7962c339c2206b164d845c1',  # bcl2fastq2-2.20.0-find-boost.patch
+]
+
+builddependencies = [
+    ('CMake', '3.21.1'),
+]
+
+dependencies = [
+    ('libxml2', '2.9.10'),
+    ('libxslt', '1.1.34'),
+    ('zlib', '1.2.11'),
+]
+
+start_dir = 'src'
+
+# check versions of libxml2 and libxslt from EB
+local_cmakelists = '%(builddir)s/bcl2fastq/src/CMakeLists.txt'
+local_sed_libver = 'sed -i "s/%(l)s_VERSION [0-9\.]*/%(l)s_VERSION $EBVERSION%(l)s/g" %(f)s && '
+preconfigopts = local_sed_libver % {'l': 'LIBXML2', 'f': local_cmakelists}
+preconfigopts += local_sed_libver % {'l': 'LIBXSLT', 'f': local_cmakelists}
+
+# remove hardcoded compilation flags
+local_cmakecxx = '%(builddir)s/bcl2fastq/src/cmake/cxxConfigure.cmake'
+preconfigopts += 'sed -i "s/-std=[a-z0-9\+]* //g;s/-O. //g" %s && ' % local_cmakecxx
+
+configopts = '-DBCL2FASTQ_VERSION:STRING=%(version)s '
+configopts += '-DBCL2FASTQ_PREFIX:STRING=%(installdir)s '
+configopts += '-DBCL2FASTQ_SOURCE_DIR:STRING=%(builddir)s/bcl2fastq/src '
+configopts += '-DBoost_NO_BOOST_CMAKE=ON '
```

Comment:
Please keep the comment about this option from the recently merged PR that added it.

Reply:
i.e. #14741

Reply:
@akesandgren I added the comment.

Reply:
btw, do you mind fixing the typo I made? should be "use any system installed"

Reply:
@akesandgren No problem, I fixed the typo.